specifying-digests.md

```diff
@@ -0,0 +1,101 @@
+# Specifying image digests
+There are a number of different ways we could specify the digests for images that another image will be built _from_.
+
+## Digest in the `FROM` tag
+As described earlier in the [Docker tag section](docker-tags.md), it is possible to specify the image digest in the `FROM` tag:
+```dockerfile
+FROM python:2.7.13-slim@sha256:ca74abc54d21a6f1f16aaa483b931fe2f536eff5b9b5e77856c61173969605d2
+```
+or...
+```dockerfile
+FROM python@sha256:ca74abc54d21a6f1f16aaa483b931fe2f536eff5b9b5e77856c61173969605d2
+```
+
+The main problem with this is that the Docker builder will ignore the tag part of the image tag (`:2.7.13-slim`). It is possible to place any string there and as long as it is a syntactically valid tag, the parser will parse it, ignore it, and consider only the digest hash.
+
+### Advantages:
+* Docker will definitely build your image using exactly the same `FROM` image every time.
+* Native Docker, not defining some new thing.
+
+### Disadvantages:
+* Tag information no longer checked. Could become incorrect.
+* Adding the digest to tags is not commonly done.
+* A bit clunky.
+
+## Digest in a "metadata" comment
+It could be possible just to add a comment to the Dockerfile with the image digest. This wouldn't be picked up by Docker itself, but a separate tool could. Most importantly, the Git repository would track the digest of the `FROM` tag.
```

Comment:
Minor phrasing issue: something like "the digest is tracked in the git repo" would make it clearer that the repo itself isn't doing any magic to track this, the maintainer is putting the information in the repo.